### ISSUE TITLE
Option to hide the expander column for expandOnRowClicked

### ISSUE THREAD

**Issue by dandv** [jbetancur/react-data-table-component#431]
**Is your feature request related to a problem? Please describe.**
I'm trying to build a mobile layout that expands each row on tap, and horizontal space is scarce.

**Describe the solution you'd like**
I would like to hide the expander column altogether.

**Describe alternatives you've considered**
I tried setting `expandableIcon` to `{} but that left the column empty, and occupying the space. Setting `customStlyes` to `expanderButton: { style: { display: 'none' } }` had the same effect.

**Additional context**
Demo [here](https://cryptoclimate.io/) until I find an alternative solution.

**Comment by jbetancur** [comment]
Sure, I can add a new prop that hides the expander row for a certain media size similar to column.hide.

**Comment by dandv** [comment]
Thanks @jbetancur! BTW, any ideas on [this related issue](https://github.com/jbetancur/react-data-table-component/issues/234#issuecomment-575005738)?

**Comment by jbetancur** [comment]
Ok this is implemented and I will release a 6.3.0 tonight. `expandableRowsHideExpander` will now hide all expanders. Of course, this only makes sense if you are using `expandOnRowClicked` or `expandOnRowDoubleClicked`.

Additionally. you can now style the expanderCell via `customSyles`

for example let's say to want to hide the expanderCell on a certain screen width:

```js
  expanderCell: {
    style: {
      '@media (max-width: 768px)': {
        display: 'none',
      },
    },
  },

```

Your expander will now be hidden when your width is below 768px

**Comment by wilsonwg** [comment]
I think this option should be part of the documentation.
